File: KeyboardInput/BMICalculator/BMICalculator.java
import java.util.Scanner;

public class BMICalculator
{
  public static void main( String[] args )
  {
    Scanner keyboard = new Scanner(System.in);
    int weight;
    double height, BMI;
    System.out.println( " Your height in m: " );
    height = keyboard.nextDouble();
    System.out.println( " Your weight in kg: " );
    weight = keyboard.nextInt();
    System.out.println();
    BMI = weight / height * height;
    System.out.println( " Your BMI is " + BMI );
  }
}

File: c++/KeyboardInput/BMICalculator/BMICalculator.cpp
#include <iostream>
using namespace std;
int main()
{
    float bmi, w, h;
    cout<<"Please enter your weight in kg: \n";
        cin>>w;
    cout<<"Please enter your height in m: \n";
        cin>>h;
    bmi=(w)/(h*h);
    cout<<"Your BMI is: \n"<<bmi<<endl;

    return 0;
}